server/api/jellyfin.ts

```diff
@@ -0,0 +1,264 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import axios, { AxiosInstance } from 'axios';
+import logger from '../logger';
+
+export interface JellyfinUserResponse {
+  Name: string;
+  ServerId: string;
+  ServerName: string;
+  Id: string;
+  PrimaryImageTag?: string;
+}
+
+export interface JellyfinLoginResponse {
+  User: JellyfinUserResponse;
+  AccessToken: string;
+}
+
+export interface JellyfinLibrary {
+  type: 'show' | 'movie';
+  key: string;
+  title: string;
+  agent: string;
+}
+
+export interface JellyfinLibraryItem {
+  Name: string;
+  Id: string;
+  HasSubtitles: boolean;
+  Type: 'Movie' | 'Episode' | 'Season' | 'Series';
+  SeriesName?: string;
+  SeriesId?: string;
+  SeasonId?: string;
+  SeasonName?: string;
+  IndexNumber?: number;
+  ParentIndexNumber?: number;
+  MediaType: string;
```

Comment:
Can this be something else than `Video` considering we are only looking at specific library types?